src/components/TradeAssetSearch/hooks/useGetPopularAssetsQuery.tsx

```diff
@@ -36,11 +37,12 @@ export const queryFn = async () => {
   // add second-class citizen chains to popular assets for discoverability
   if (enabledFlags.HyperEvm) assetIds.push(hyperEvmAssetId)
   if (enabledFlags.Monad) assetIds.push(monadAssetId)
+  if (enabledFlags.Near) assetIds.push(nearAssetId)
   if (enabledFlags.Plasma) assetIds.push(plasmaAssetId)
   if (enabledFlags.Tron) assetIds.push(tronAssetId)
   if (enabledFlags.Sui) assetIds.push(suiAssetId)
 
-  for (const assetId of assetIds) {
+  for (const assetId of [...new Set(assetIds)]) {
```

Comment:
We were getting dupe assets in popular assets i.e if an asset is both part of pushed set and base set